### ISSUE TITLE
[$40][MSFT-222] Challenge feed page: colour contrast issues.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2881]
**Steps to Reproduce**
1. Go to the Challenges page: https://www.topcoder.com/challenges. 
2. Please run the automated tool. 
3. Capture the colour contrast issues.

**Expected Result**
The colour contrast should comply as per the WCAG minimum contrast requirements.

**Actual Result**
The colour contrast between the background and foreground of the elements doesn't comply with WCAG minimum contrast requirements.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: AA
**WCAG Success Criterion**: 1.4.3 Contrast (minimum)
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2037

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096020 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096020 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096020 has been updated - it has been assigned to Timor.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by romitgit** [comment]
PR for this issue : https://github.com/topcoder-platform/community-app/pull/2993

Dependent PR
PR for topcoder-react-ui-kit : https://github.com/topcoder-platform/topcoder-react-ui-kit/pull/15

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096765 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
@romitgit, please sign-up with Topcoder X tool<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096765 has been updated - it has been assigned to Timor.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by romitgit** [comment]
PR for this issue : https://github.com/topcoder-platform/community-app/pull/3062

Dependent PR
PR for topcoder-react-ui-kit : https://github.com/topcoder-platform/topcoder-react-ui-kit/pull/15

**Comment by tosha5252** [comment]
@sushilshinde What tool is being used to check color contrast issues?  I'm checking with the WAVE tool and there are still many issues with the page...also I can see there are some buttons that don't have the changed blue color, which should be #006AD7 for all blue button background and text.

**Comment by sushilshinde** [comment]
@tosha5252 please ask on the slack

**Comment by romitgit** [comment]
Whats the status?

**Comment by sushilshinde** [comment]
> @sushilshinde What tool is being used to check color contrast issues? I'm checking with the WAVE tool and there are still many issues with the page...also I can see there are some buttons that don't have the changed blue color, which should be #006AD7 for all blue button background and text.

@tosha5252 Can you share screenshots?

**Comment by lakshmiathreya** [comment]
@sushilshinde this is a generic Issue - we need a list of fixes that actually went into this Issue fix. Currently WAVE shows 14 contrast errors.

**Comment by lakshmiathreya** [comment]
Moving to QA Pass based on overall progress on contrast - however, this will be an ongoing exercise in fixing contrast. Consensus with @Glenn @sushilshinde @nkumar-topcoder - this is a generic issue on contrast - cannot be verified specifically.

**Comment by lakshmiathreya** [comment]
Closing this one based on comparative progress made overall in resolving contrast issues - however its not complete - similar observations as in #2996 - several outstanding contrast issues should be reviewed as part of that Issue - currently 'On Hold'.
Based on the observation, closing this issue.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30096765<br/><br/>```This is an automated message for crazyk via Topcoder X```
